search-app/docs/api.md

```diff
@@ -458,9 +458,15 @@ __Example__
 
 #### <a name="csv"></a> CSV
 
-The comma separated value (CSV) format is only supported for granules.
+The comma separated value (CSV) format is supported for collections and granules.
 
-__Example__
+__Examples__
+
+```csv
+Data Provider,Short Name,Version,Entry Title,Processing Level,Platforms,Start Time
```

Comment:
nit: make it clear that this is the collection csv response, and the other one is the granule csv response.